src/Microsoft.OpenApi.Readers/YamlConverter.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Nodes;
+using Json.More;
+using SharpYaml;
+using SharpYaml.Serialization;
+
+namespace Microsoft.OpenApi.Readers
+{
+    /// <summary>
+    /// Provides extensions to convert YAML models to JSON models.
+    /// </summary>
+    public static class YamlConverter
+    {
+        /// <summary>
+        /// Converts all of the documents in a YAML stream to <see cref="JsonNode"/>s.
+        /// </summary>
+        /// <param name="yaml">The YAML stream.</param>
+        /// <returns>A collection of nodes representing the YAML documents in the stream.</returns>
+        public static IEnumerable<JsonNode> ToJsonNode(this YamlStream yaml)
+        {
+            return yaml.Documents.Select(x => x.ToJsonNode());
```

Comment:
Evidence would suggest otherwise
https://github.com/microsoft/OpenAPI.NET/blob/df1b405e537d0f3c2a829b6a80fef82959d0eab5/src/Microsoft.OpenApi/Models/OpenApiOperation.cs#L286

Reply:
Sure, okay.  I'm not worried about these nuances right now.  Let's please focus on making it work.